base_technical_features_adj/README.rst

```diff
@@ -22,6 +22,15 @@ Technical features group adjustments
 This module limits the visibility of the 'technical_features' attribute
 in the preference view, making it accessible exclusively to internal users.
 
+Background
+~~~~~~~~~~
+
+In the SST environment, backend access is currently allowed for non-internal users.
```

Comment:
@AungKoKoLin1997 Avoid making the module description customer specific, especially the ones in public repos.  This explanation can be much shorter as well (avoid verbosity).